scripts/sysman/powerLimits.yml

```diff
@@ -108,6 +114,7 @@ details:
     - "The same limit can be supplied multiple times. Limits are applied in the order in which they are supplied."
     - "The application may call this function from simultaneous threads."
     - "The implementation of this function should be lock-free."
+    - "1.6": "Note: This function is deprecated."
```

Comment:
It would be good the explicitly state for all of these what can be used as the replacement, or state that "There is no replacement", when there is none.